src/util/Constants.js

```diff
@@ -205,6 +205,7 @@ const Endpoints = exports.Endpoints = {
     return {
       Emoji: emojiID => `${root}/emojis/${emojiID}.png`,
       Asset: name => `${root}/assets/${name}`,
+      DefaultAvatar: number => `${root}/embed/avatars/${number}.png`,
```

Comment:
The links go from `cdn.discordapp.com/embed/avatars/0.png` to `cdn.discordapp.com/embed/avatars/4.png`